### PR TITLE
expose public pulling image repository

### PR DESCRIPTION
**Pull request by juanvallejo**
Fixes https://github.com/openshift/origin-web-console/issues/1958

cc @spadgett

### CODE REVIEW

File: views/imagestream-body.html [hunk 1]
```diff
@@ -36,6 +36,8 @@
     <dd ng-if-end><tt>{{imagestream.spec.dockerImageRepository}}</tt></dd>
     <dt>Pulling repository</dt>
     <dd><tt>{{imagestream.status.dockerImageRepository}}</tt></dd>
+    <dt ng-if-start="imagestream.status.publicDockerImageRepository">Public pulling repo</dt>
+    <dd ng-if-end><tt>{{imagestream.status.publicDockerImageRepository}}</tt></dd>
```
Comment (juanvallejo) [inline views/imagestream-body.html:40]: @spadgett not sure if it'd be better to leave this out if there is no `publicDockerImageRepository` field, or simply state that there is none / instructions on how expose a registry?

**Comment by spadgett** [comment]
@juanvallejo Thanks. You'll need to open against https://github.com/cockpit-project/registry-image-widgets

@petervo

**Comment by juanvallejo** [comment]
@spadgett re-opened PR here: https://github.com/cockpit-project/registry-image-widgets/pull/10

**Comment by spadgett** [comment]
Thanks, closing in favor of cockpit-project/registry-image-widgets#10

**Pull request closed without merging**